desktop/windows/index.md

```diff
@@ -142,26 +142,27 @@ containers. Alternatively, you can opt not to share it by selecting **Cancel**.
 
 #### Proxies
 
-Docker Desktop detects the HTTP/HTTPS proxy settings and
-automatically propagates these to Docker. For example, if you set your proxy
-settings to `http://proxy.example.com`, Docker uses this proxy when pulling containers.
-
-Your proxy settings, however, will not be propagated into the containers you start.
-If you wish to set the proxy settings for your containers, you need to define
-environment variables for them, just like you would do on Linux, for example:
-
-```ps
-> docker run -e HTTP_PROXY=http://proxy.example.com:3128 alpine env
-
-PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
-HOSTNAME=b7edf988b2b5
-TERM=xterm
-HOME=/root
-HTTP_PROXY=http://proxy.example.com:3128
-```
 
-For more information on setting environment variables for running containers,
-see [Set environment variables](/engine/reference/commandline/run/#set-environment-variables--e---env---env-file).
+HTTP/HTTPS proxies can be used when
```

Comment:
For lines 146-160: Same comments as for Mac

Reply:
Updated the Mac section.